client/src/assets/images/components/bear-progress-modal.tsx

```diff
@@ -11,8 +11,13 @@ function BearProgressModal(
         width='187'
         height='178'
         fill='none'
+        role='img'
+        aria-labelledby='bear-progress-label'
         {...props}
       >
+        <title id='bear-progress-label'>
+          Illustration of a bear holding a jar with hearts in the background
```

Comment:
```suggestion
          Illustration of an adorable, but sad looking teddy bear holding an empty money jar.
```

Granted, alt text is an art, not a science, so there is no one correct solution here. I do think we should try to describe the purpose of this image. To me it is to tug on the hearts strings just a little, implying that fCC needs their donation.

Perhaps "sad" isn't the best word to describe the look on the bear's face? Please suggest alternatives if you have them.

Reply:
Maybe `an adorable teddy bear with a pleading expression`?  For reference in case anyone else has suggestions these are the two illustrations that need alts:
<img width="364" alt="Screenshot 2023-09-26 at 1 13 10 PM" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/13138083/60c50b60-5cec-4b8a-9f4a-13162d625527">

Reply:
Will do, thank you. What is the process for adding new translation strings? I saw in guidelines only English should be added, but is there anywhere else I'd need to do in order to get them into to the translation queue?

Reply:
They'll be synced to the translation platform once merged.

Reply:
@bbsmooth thanks again for the feedback, I've updated the alts for the bear illustrations to be:
"bear-progress-alt": "Illustration of an adorable teddy bear with a pleading expression holding an empty money jar.",
"bear-completion-alt": "Illustration of an adorable teddy bear holding a large trophy."